CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ IMPROVEMENTS:
 
 BUG FIXES:
 
+* resource/cloudflare_record: Ensure `Create`/`Update`/`Read` functions all apply the record transformation and store the correct state values [GH-217]
```

Comment:
Not a big deal, but we usually modify Changelog _after_ merging PRs, via the GitHub UI (pencil icon). It helps us to avoid conflicts in case we merge other PRs whilst one is being reviewed.